### PR TITLE
getAll relations objects for the type

### PR DESCRIPTION
**Pull request by RohiniKambam**
1.Created Types.
2.Created  relationships between types.
3.Using that relationship getAll the relations.
[getAllrelationdata.txt](https://github.com/PogeyanOSS/cmis-swagger/files/2282044/getAllrelationdata.txt)

### CODE REVIEW

File: copper-swagger/src/main/java/com/pogeyan/swagger/server/ApiDocsServlet.java [hunk 1]
```diff
@@ -128,16 +128,20 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response, S
 			JSONObject obj = null;
 			ContentStream stream = null;
 			String repositoryId = pathFragments[0];
-			String typeId = pathFragments[1].replace("_", ":");
+			// String typeId = pathFragments[1].replace("_", ":");
+			String typeId = pathFragments[1];
```
Comment (azarMohamed) [inline copper-swagger/src/main/java/com/pogeyan/swagger/server/ApiDocsServlet.java]: Check replace method need r not

File: copper-swagger/src/main/java/com/pogeyan/swagger/server/ApiDocsServlet.java [hunk 2]
```diff
@@ -128,16 +128,20 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response, S
...
 			String id = pathFragments[2];
 
 			String select = null;
 			String filter = null;
 			String order = null;
 			if (request.getQueryString() != null) {
-				select = request.getParameter("select").replace("_", ":");
-				filter = request.getParameter("filter").replace("_", ":");
-				order = request.getParameter("orderby").replace("_", ":");
+				select = request.getParameter("select") != null ? request.getParameter("select").replace("_", ":")
```
Comment (azarMohamed) [inline copper-swagger/src/main/java/com/pogeyan/swagger/server/ApiDocsServlet.java]: code format not correct

File: copper-swagger/src/main/java/com/pogeyan/swagger/services/SwaggerApiService.java [hunk 3]
```diff
@@ -666,8 +669,7 @@ public static JSONObject invokeGetTypeDefMethod(String repositoryId, String type
 		JSONObject obj = JSONConverter.convert(typedef, DateTimeFormat.SIMPLE);
 		if (includeRelationship) {
 			List<FileableCmisObject> relationType = SwaggerHelpers.getRelationshipType(session, typeId);
-			JSONArray childJson = getRelationshipChild(session, relationType, JsonArray);
-			obj.put("relations", childJson);
+			JSONArray childJson = null;
```
Comment (azarMohamed) [inline copper-swagger/src/main/java/com/pogeyan/swagger/services/SwaggerApiService.java]: check this

File: copper-swagger/src/main/java/com/pogeyan/swagger/services/SwaggerApiService.java [hunk 4]
```diff
@@ -843,15 +847,67 @@ public static JSONObject invokeGetAllMethod(String repositoryId, String type, St
...
+
 		}
 
 		for (CmisObject child : children.getPage()) {
 			Map<String, Object> propmap = compileProperties(child, session);
+			if (includeRelationship) {
+				List<FileableCmisObject> relationType = SwaggerHelpers.getRelationshipType(session, type);
+				JSONArray childArray = new JSONArray();
+				for (FileableCmisObject ob : relationType) {
+					Object typeObj1 = ob.getPropertyValue("target_table");
```
Comment (azarMohamed) [inline copper-swagger/src/main/java/com/pogeyan/swagger/services/SwaggerApiService.java]: Give proper names assign to variables

File: copper-swagger/src/main/java/com/pogeyan/swagger/services/SwaggerApiService.java [hunk 5]
```diff
@@ -843,15 +847,67 @@ public static JSONObject invokeGetAllMethod(String repositoryId, String type, St
...
+
+				propmap.put("relation", childArray);
+			}
 			json.put(child.getName(), propmap);
 		}
+
 		return json;
 	}
 
+	private static Object JsonArray() {
```
Comment (azarMohamed) [inline copper-swagger/src/main/java/com/pogeyan/swagger/services/SwaggerApiService.java]: Remove unwanted code

**Comment by RohiniKambam** [comment]
1.Created Types.
2.Created relationships between types.
3.Using that relationship getAll the relations.
[getAllrelationdata.txt](https://github.com/PogeyanOSS/cmis-swagger/files/2282822/getAllrelationdata.txt)

**Pull request closed without merging**